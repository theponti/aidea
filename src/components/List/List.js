import React from 'react'
import t from 'prop-types'

import ListItem from '../ListItem'
import styles from './List.module.scss'

List.propTypes = {
  data: t.arrayOf(
    t.shape({
      description: t.string
    })
  )
}
export function List ({ data }) {
  return (
    <div className={styles.container}>
      {data.map((item) => (
        <ListItem key={item._id} item={item} />
      ))}
    </div>
  )
}

export default List
