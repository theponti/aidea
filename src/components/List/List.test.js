import React from 'react'
import { render } from '@testing-library/react'
import List from './List'

describe('<List/>', () => {
  it('should render component', () => {
    const { container } = render(<List data={[]} />)
    expect(container).toMatchSnapshot()
  })
})
