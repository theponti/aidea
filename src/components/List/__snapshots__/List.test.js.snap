// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<List/> should render component 1`] = `
<div>
  <div
    class="container"
  />
</div>
`;
